lib/transactional.js

```diff
@@ -1,23 +1,38 @@
 'use strict';
 let _ = require('lodash')
   , translator = require('../lib/translator')
-  , version = require('../package.json').version,
-  rules = [
+  , version = require('../package.json').version
+  , rules = [
+    [/<%asm_group_unsubscribe_url%>/, '{{asm_group_unsubscribe_url}}'],
```

Comment:
spacing after/before curly braces (as we're doing for all variables)
`'{{ asm_group_unsubscribe_url }}'